src/networking.c

```diff
@@ -1265,9 +1266,8 @@ int clientHasPendingReplies(client *c) {
             c->ref_block_pos == tail->used) return 0;
 
         return 1;
-    } else {
-        return c->bufpos || listLength(c->reply);
     }
+    return c->bufpos || listLength(c->reply);
```

Comment:
is this change necessary?

Reply:
not really :) removing it to keep the focus of a single change on this PR